crates/bevy_macro_utils/src/lib.rs

```diff
@@ -86,6 +86,36 @@ impl BevyManifest {
     pub fn parse_str<T: syn::parse::Parse>(path: &str) -> T {
         syn::parse(path.parse::<TokenStream>().unwrap()).unwrap()
     }
+
+    pub fn encase_path() -> syn::Path {
+        const BEVY: &str = "bevy";
+        const BEVY_RENDER: &str = "bevy_render";
+        const ENCASE: &str = "encase";
+
+        let bevy_manifest = BevyManifest::default();
+        bevy_manifest
+            .maybe_get_path(BEVY)
+            .map(|bevy_path| {
+                let mut segments = bevy_path.segments;
+                segments.push(BevyManifest::parse_str("render"));
+                segments.push(BevyManifest::parse_str(ENCASE));
+                syn::Path {
+                    leading_colon: None,
+                    segments,
+                }
+            })
+            .or_else(|| bevy_manifest.maybe_get_path(BEVY_RENDER))
+            .map(|bevy_render_path| {
```

Comment:
This second map needs to be inside the or_else, otherwise it gets appended to the end of the first code path above using BEVY.

Reply:
Indeed, thanks!